topi/python/topi/generic/extern.py

```diff
@@ -35,8 +36,5 @@ def schedule_extern(outs):
     sch: Schedule
         The computation schedule for the op.
     """
-    target = tvm.target.current_target(allow_none=False)
-    if target.target_name != "llvm":
-        raise RuntimeError("schedule_extern not registered for '%s'" % target)
-    outs = [outs] if isinstance(outs, tvm.tensor.Tensor) else outs
-    return tvm.create_schedule([x.op for x in outs])
+    target = tvm.target.current_target()
+    return cpp.generic.schedule_extern(target, outs)
```

Comment:
Do we need to pass "target" variable to c++ schedule_extern?

Reply:
hey @soiferj can you address this comment ...

Reply:
Answered below